Rearranged LevelScene with Level Buttons

Deumlich-S0573093 opened the pull request:
rearranged the levelScene

Deumlich-S0573093 committed:
loaded a new asset (level button) level_tile.png
rearranged the levelScene

mango1603 commented:
Can you make the click area larger, now it only works when I click on the number, maybe the whole button

Deumlich-S0573093 commented:
> Can you make the click area larger, now it only works when I click on the number, maybe the whole button

Sure, will do that!

Deumlich-S0573093 committed:
changed the button style for level selection (cloud)
created a container that holds the font and the button asset
fixed the issue of beeing able to click on the button (not only the font) to select the level

Deumlich-S0573093 committed:
removed bestScore scene
added flag that shows if a level was completed before
adds Best Score text in LevelsScene for each level

Deumlich-S0573093 committed:
removed bestScore scene
added flag that shows if a level was completed before
adds Best Score text in LevelsScene for each level

Deumlich-S0573093 committed:
removed bestScore scene
added flag that shows if a level was completed before
adds Best Score text in LevelsScene for each level

NadyaKan committed:
Merge branch 'master' into Rearrange_LevelScene
